data/human/hails.txt

```diff
@@ -1964,6 +1964,79 @@ phrase "friendly civilian"
 	word
 		"."
 
+phrase "friendly civilian"
+	word
+		"Sometimes it's too good to be true."
+		"Don't take unnecessary risks."
+		"Crime doesn't pay in the end."
+		""
+	word
+		" It's a fact that "
+		" I've heard "
+		" I heard a rumor that "
+	word
+		"smugglers almost always get caught in the end."
+		"even the most powerful warship won't save you from a knife in the back."
+		"the Quarg will arrest you if they catch a whiff of any of their equipment on your ship."
+		"the Quarg imprison anyone they catch carrying their outfits."
+		"nine out of ten pirates die within their first year."
+
+phrase "friendly civilian"
+	word
+		"When I was younger,"
+		"When I was a kid,"
+		"When I was your age,"
+		"Long ago,"
+	word
```

Comment:
Instead of just `dreamed of`, another phrase could be `always wanted to`, `always wished to`, `fancied`, `thought about`, et c.

Reply:
I've added `fancied`, and `thought about` to that one, although the rest wouldn't quite fit and I don't want to have to make a whole new phrase for it.